fullscreen-basic/definition.json

```diff
@@ -381,15 +381,15 @@
           "font_size": 17,
           "font_color": "#FFFFFF",
           "background": "#5f7384",
-          "deeplink_url": "myapp://path/to/content or http://your.url"
+          "placeholder": "myapp://path/to/content or http://your.url"
         },
         {
           "text": "Button 2",
           "font_family": "Open Sans",
           "font_size": 17,
           "font_color": "#5f7384",
           "background": "#FFFFFF",
-          "deeplink_url": "myapp://path/to/content or http://your.url"
+          "placeholder": "myapp://path/to/content or http://your.url"
```

Comment:
This is wrong. Here you need to keep the key `deeplink_url`, since that is the name of the field. And you need to put the value empty here too, since this is the default that will appear in the editor and if it has this as default it will break the template unless the user clears it up.